README.md

```diff
@@ -131,14 +137,17 @@ export.status
 
 ### Panels
 
-To get all the panels:
+To paginate all the panels:
 
 ```ruby
-QualtricsAPI.panels.fetch
+panels = QualtricsAPI.panels.fetch
+# => #<QualtricsAPI::PanelCollection:0x007f8769aae2c0 ....>
+
+panels = panels.next # get next page if not panels.last?
```

Comment:
can we rename it to something like `next_page` ? 
`next` sounds like it's fetching a single record.